common/api/core-common.api.md

```diff
@@ -5727,6 +5731,15 @@ export interface MapLayerKey {
 // @public
 export type MapLayerProps = ImageMapLayerProps | ModelMapLayerProps;
 
+// @beta (undocumented)
+export type MapLayerProviderArrayProperty = Array<number | string | boolean>;

```

Comment:
This type (used in MapLayerProviderProperties and compareSimpleArrays) - are you sure you didn't mean `number[] | string[] | boolean[]`? The way you have it now permits things like `{ array: ["abc", 123, true] }`.

Reply:
I prefer the strict approach, so I changed it to `number[] | string[] | boolean`